GitUI/Avatars/AvatarMemoryCache.cs

```diff
@@ -21,6 +21,8 @@ public AvatarMemoryCache(IAvatarProvider inner, int capacity = 30)
             _cache = new MruCache<(string email, int imageSize), Image>(capacity);
         }
 
+        public bool IsFast => true;
```

Comment:
![image](https://github.com/gitextensions/gitextensions/assets/4403806/c4a65970-9ba3-4bce-ab09-d81e3238be62)

I _think_ I know what this is for, and if I'm not wrong then perhaps it'd be better if we'd call it it 
```suggestion
        public bool PerformsIo => false;
```